### PR TITLE
[CI] Start using manifest to simplify the pwsh import.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

File: tools/devops/automation/scripts/MaciosCI.psd1 [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+GUID = 'a3293c8a-ffbf-4b14-90fe-72ae1b623473'
+
+# Author of this module
+Author = 'mandel'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corp.'
+
+# Copyright statement for this module
+Copyright = '(c) mandel. All rights reserved.'
```
Comment (dalexsoto) [inline tools/devops/automation/scripts/MaciosCI.psd1:30]:
```suggestion
Copyright = 'Copyright (C) Microsoft Corporation. All Rights Reserved.'
```

**Review by dalexsoto — APPROVED** [review]

File: tools/devops/automation/scripts/MaciosCI.psd1 (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/scripts/MaciosCI.psd1:30]: Es mio

File: tools/devops/automation/templates/build/upload-azure.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ steps:
     allowFailedBuilds: true
     path: $(Build.SourcesDirectory)/artifacts
 
-- powershell : |
+- pwsh: |
```
Comment (tj-devel709) [inline tools/devops/automation/templates/build/upload-azure.yml:24]: These mean the same thing right?

Comment (tj-devel709) [reply tools/devops/automation/templates/build/upload-azure.yml:24]: Oh wait I see this 
`powershell runs Windows PowerShell and will only work on a Windows agent. pwsh runs PowerShell Core, which must be installed on the agent or container.`
Can you explain to me the difference?

**Pull request merged**